### ISSUE TITLE
Overlaping

### ISSUE THREAD

**Issue by aleksnadar5** [minhur/bootstrap-toggle#45]
I have downloaded bootstrap-toggle from Nuget and have tried one of the simple examples, the buttons are overlapped, using MVC 5.0, anyone else have the same issue?

@section Scripts {
![untitled](https://cloud.githubusercontent.com/assets/11447402/7238490/164d761e-e7a3-11e4-8bf0-6839af52c914.png)

<input id="toggle-event" data-toggle="toggle" type="checkbox">
<div id="console-event"></div>
<script>
    $(function () {
        $('#toggle-event').change(function () {
            $('#console-event').html('Toggle: ' + $(this).prop('checked'))
        })
    })
</script>
}

![Uploading Untitled.png . . .]()

**Comment by aleksnadar5** [comment]
There is no problem if I use the js and css by downloading them so there is something wrong with the version on Nuget

**Comment by gomathi04** [comment]
I think  you have missed the class = "toggle"  in <input id="toggle-event" data-toggle="toggle" type="checkbox">

**Comment by jklaphake** [comment]
Yes, I have got the same problem. I have taken a very simple example:
<input checked data-toggle="toggle" type="checkbox">
I have downloaded the js and css:
bootstrap-toggle.min.css
bootstrap-toggle.min.js

I don't quite understand the previous comment about missing the class="toggle" ??
I did add it to the input, but that made no difference
